src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -303,6 +303,7 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
 		CacheFile="$(_AndroidLibraryProjectImportsCache)"
 		DesignTimeBuild="$(DesignTimeBuild)"
 		Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
+		AarLibraries="@(AndroidAarLibrary)"
```

Comment:
Don't forget to add documentation to `Documentation/build_process.md` for the `@(AndroidAarLibrary)` build action. :-)

Reply:
Here again, we don't have documentation for anything about bindings at all.

Reply:
I will add the docs for this.. but I think binding docs will need to be a different PR. We should probably look at grabbing the docs from the docs team on that.

Reply:
IMO product documentation should NOT be maintained within this repo but should be maintained at docs team. Having duplicate effort makes brings not only waste of time but also bad inconsistency.

Reply:
@atsushieno: the docs team *does not* maintain low-level build system documentation. *We* do.

It's not a duplication of effort to update `Documentation/build_process.md`. It's the *only* location for such documentation.

Reply:
That's weird. I did create PR yesterday: documentation/pull/2871